src/StorageSync/StorageSync.Test/StorageSync.Test.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.StorageSync" Version="3.0.0" />
+    <PackageReference Include="Microsoft.Azure.Management.StorageSync" Version="3.1.0" />
```

Comment:
Need regression tests for the new cmdlet

Reply:
> Need regression tests for the new cmdlet

Pending...

Reply:
Done